src/sku/molecule/domain/src/snapshots/molecule_dataset_snapshots.rs

```diff
@@ -164,6 +164,7 @@ impl MoleculeDatasetSnapshots {
         DatasetSnapshots::versioned_file(
             alias,
             vec![
+                // TODO: NOT NULL cols?
```

Comment:
That would make the change breaking, we only support adding optional columns for automatic schema migration

Reply:
In the case of a Molecule versioned file, yes

But in that case, it turns out I need to remove the strict schemas (`SetDataSchema`) that I added earlier